src/mixins/I13nMixin.js

```diff
@@ -216,6 +216,21 @@ var I13nMixin = {
     },
 
     /**
+     * _shouldFollowLink, provide a hook to check followLink.
+     * It check if component implement its own shouldFollowLink() method, 
+     * otherwise return props.followLink or props.follow
+     * @method _shouldFollowLink
+     * @private
+     */
+    _shouldFollowLink: function () {
```

Comment:
The reason I put it in mixin is to leave document for users to know they can implement `shouldFollowLink()` method.